test/page.spec.js

```diff
@@ -130,6 +131,25 @@ describe('Page', function () {
     });
   });
 
+  describe('removing and adding event handlers', () => {
+    it('should correctly fire event handlers as they are added and then removed', async () => {
+      const { page, server } = getTestState();
+
+      const handler = sinon.spy();
+      page.on('response', handler);
+      await page.goto(server.EMPTY_PAGE);
+      expect(handler.callCount).toBe(1);
+      page.off('response', handler);
+      await page.goto(server.EMPTY_PAGE);
+      // Still one because we removed the handler.
+      expect(handler.callCount).toBe(1);
+      page.on('response', handler);
+      await page.goto(server.EMPTY_PAGE);
+      // Two now because we added the handler back.
+      expect(handler.callCount).toBe(2);
+    });
+  });
```

Comment:
a user reported a bug in 3.2 where this code didn't work. I don't think it was a bug in Mitt, but I wanted to add a test that explicitly listened to an event, stopped, and then started again just to make sure. It's a useful test to have IMO.